plugins/CoreAdminHome/templates/optOut.twig

```diff
@@ -4,31 +4,34 @@
     <meta charset="utf-8">
 </head>
 <body>
-{% if not trackVisits %}
-    {{ 'CoreAdminHome_OptOutComplete'|translate }}
-    <br/>
-    {{ 'CoreAdminHome_OptOutCompleteBis'|translate }}
+{% if dntFound %}
+    {{ 'CoreAdminHome_OptOutDntFound'|translate }}
 {% else %}
-    {{ 'CoreAdminHome_YouMayOptOut'|translate }}
-    <br/>
-    {{ 'CoreAdminHome_YouMayOptOutBis'|translate }}
+	{% if not trackVisits %}
+		{{ 'CoreAdminHome_OptOutComplete'|translate }}
+		<br/>
+		{{ 'CoreAdminHome_OptOutCompleteBis'|translate }}
+	{% else %}
+		{{ 'CoreAdminHome_YouMayOptOut'|translate }}
+		<br/>
+	{{ 'CoreAdminHome_YouMayOptOutBis'|translate }}
+	{% endif %}
+	<br/><br/>
+	<form method="post" action="?module=CoreAdminHome&amp;action=optOut{% if language %}&amp;language={{ language }}{% endif %}">
+		<input type="hidden" name="nonce" value="{{ nonce }}" />
+		<input type="hidden" name="fuzz" value="{{ "now"|date }}" />
+		<input onclick="this.form.submit()" type="checkbox" id="trackVisits" name="trackVisits" {% if trackVisits %}checked="checked"{% endif %} />
+		<label for="trackVisits"><strong>
+		{% if trackVisits %}
+			{{ 'CoreAdminHome_YouAreOptedIn'|translate }} {{ 'CoreAdminHome_ClickHereToOptOut'|translate }}
+		{% else %}
+			{{ 'CoreAdminHome_YouAreOptedOut'|translate }} {{ 'CoreAdminHome_ClickHereToOptIn'|translate }}
+		{% endif %}
+		</strong></label>
+		<noscript>
+			<button type="submit">{{ 'General_Save'|translate }}</button>
+		</noscript>
+	</form>
```

Comment:
This is a minor thing but the indentation has been replaced by tabs in this file, whereas we use spaces. Could you please replace the indentation?